api/geth_backend.go

```diff
@@ -882,26 +882,7 @@ func (b *GethStatusBackend) AppStateChange(state string) {
 	b.log.Info("App State changed", "new-state", s)
 	b.appState = s
 
-	if s == appStateForeground && !b.forceStopWallet {
```

Comment:
Wallet service is not started on foreground on `status-go` side anymore, it leaves a client side opportunity to decide whether new blocks should be watched.